source/deployment/advanced-permissions.rst

```diff
@@ -130,6 +141,30 @@ Example: In Team C, restrict public channel creation to Admins. As the default f
   ii. Select **Add Teams** to add Team B to the **Select teams to override permissions** list, locate Team B, and choose **Add**.   
   iii. In the **All Members** panel, in the **Manage Public Channels** section, uncheck the box for **Create Channels**.
   iv. In the **Team Administrators** panel, in the **Manage Public Channels** section, check the box for **Create Channels**.
+  
+Read only channels
+~~~~~~~~~~~~~~~~~~
+
+**Configure a channel so that members can post/reply/react but guests can only read and react.**
+
+1. Navigate to **System Console > User Management > Channels** (or **System Console > Advanced Permissions** in versions prior to 5.12).
```

Comment:
This feature wasn't available prior to v5.12 (it is being release in v5.22)  so the "... in versions prior to 5.12" doesn't apply.